src/functions-and-arrays.js

```diff
@@ -52,14 +123,45 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
-
+function uniquifyArray(wordsUnique) {
+
+  let newArray = []
+  
+  if (!wordsUnique.length) {
+    return null
+  }
+
+  for (let i=0; i < wordsUnique.length; i++) {
+            
+    if (newArray.includes(uniquifyArray[i])) {
+      continue;
+    }
+
+    else {
+      newArray.push(wordsUnique[i])
+    }
+    return newArray
```

Comment:
Daaammn. Este return dentro del for loop... 💀
Está bien planteado salvo por estas 2 cosas. Fíjate mucho en estas cosas porque te pasa a menudo, tía.